.github/workflows/pr-labeler.yml

```diff
@@ -0,0 +1,13 @@
+# https://github.com/actions/labeler
+
+name: "Pull Request Labeler"
+on:
+- pull_request
+
+jobs:
+  triage:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/labeler@v3
+      with:
+        repo-token: ${{ secrets.VTOOLS_GITHUB_API_TOKEN }}
```

Comment:
Let's use `${{ secrets.GITHUB_TOKEN }}` (the token generated specifically for each action run) since the action supports it.